### ISSUE TITLE
Trouble installing dataverse R package

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse-client-r#11]
Hi! At https://github.com/IQSS/dataverse/issues/2700#issuecomment-273946411 @monogan reported some trouble getting the dataverse package installed on Windows and I was thinking that the readme of this repo would explain the best way to get support but it's non-obvious to me. Can this be clarified in the readme? Thanks!

p.s. If @monogan could get some assistance that would be great as well!

**Comment by leeper** [comment]
I think this is an installer problem (i.e., an issue in the **ghit** package that I'm working on. Temporary fix is either:

```R
devtools::install_github("IQSS/dataverse-client-r")
# or
ghit::install_github("IQSS/dataverse-client-r", build_vignettes = FALSE)
```

Let me know if that doesn't work, @monogan.

**Comment by monogan** [comment]
Hi Thomas @leeper. Unforutnately, that's still not working. I tried each, and you can see the error messages below afterwards. Any other ideas? Could it be something going on with my computer? I can't use my regular workspace right now, as my MacBook Air's OS is too out of date to update R (or even to update the OS), so I'm stuck with a Windows machine that's provided at my flat. More than likely it's 100% on my end. Thank you for all of your help.

> devtools::install_github("IQSS/dataverse-client-r")
Downloading GitHub repo IQSS/dataverse-client-r@master
from URL https://api.github.com/repos/IQSS/dataverse-client-r/zipball/master
Installing dataverse
"C:/Users/COX'SG~1/DOCUME~1/R/R-33~1.2/bin/x64/R" --no-site-file --no-environ  \
  --no-save --no-restore --quiet CMD INSTALL "C:/Users/Cox's  \
  Ground/AppData/Local/Temp/Rtmpw9u7u6/devtools49417c23ee/IQSS-dataverse-client-r-bd8d7c1"  \
  --library="C:/Users/Cox's Ground/Documents/R/R-3.3.2/library"  \
  --install-tests 

* installing *source* package 'dataverse' ...
Warning in as.POSIXlt.POSIXct(x, tz) : unknown timezone 'Europe/London'
Warning in as.POSIXlt.POSIXct(x, tz) : unknown timezone 'GMT'
Warning in as.POSIXlt.POSIXct(x, tz) :
  unknown timezone 'America/New_York'
Warning in as.POSIXlt.POSIXct(x, tz) : unknown timezone 'GMT'
Warning in as.POSIXlt.POSIXct(x, tz) :
  unknown timezone 'America/New_York'
** R
** inst
** tests
** preparing package for lazy loading
Error in if (file.size(codeFile) == file.size(loaderFile)) warning("package seems to be using lazy loading already") else { : 
  missing value where TRUE/FALSE needed
ERROR: lazy loading failed for package 'dataverse'
* removing 'C:/Users/Cox's Ground/Documents/R/R-3.3.2/library/dataverse'
Error: Command failed (1)


> ghit::install_github("IQSS/dataverse-client-r", build_vignettes = FALSE)
Error in build_and_insert(p$pkgname, d, vers, build_args, verbose = verbose) : 
  Package build for dataverse failed!
In addition: Warning message:
running command '"C:/Users/COX'SG~1/DOCUME~1/R/R-33~1.2/bin/x64/R" CMD build  --no-build-vignettes C:\Users\COX'SG~1\AppData\Local\Temp\Rtmpw9u7u6\dataverse-client-r4947786fa1' had status 1

**Comment by leeper** [comment]
What version of R do you have?

**Comment by monogan** [comment]
@leeper I've got R 3.3.2. I tried this on both 32 and 64 bit.

**Comment by leeper** [comment]
@monogan I've never seen this before, so I'm firing up the bat signal: https://twitter.com/thosjleeper/status/823836088712228864

**Comment by monogan** [comment]
@leeper Thank you. This is so strange; I have no idea what's going on. My apologies for seeming to be a "problem magnet" on this one.

**Comment by leeper** [comment]
@monogan No problem. Consensus seems to be that you should uninstall and reinstall R, as something seems to be fairly fundamentally broken.

**Comment by monogan** [comment]
@leeper @pdurbin @djbrooke @mheppler I am happy to report that I was now able to get @leeper 's code to work well, as Thomas posted here (https://github.com/IQSS/dataverse/issues/2700#issuecomment-274348230).

In case the backstory is interesting: Following Thomas & company's advice, I tried reinstalling R, at which point I learned that I only had partial administrative control over this computer. Apparently that wasn't sufficient for installing Thomas's "dataverse" package. With full administrative control, I was finally able to get it going, once I got all of the dependencies loaded. So, yes, @leeper 's code works, and nothing has changed with the switch to new Dataverse. Thank you for your patience with me and for all of your assistance.
